### ISSUE TITLE
Group Voting vs. Transacting into logical user journeys

### ISSUE THREAD

**Issue by jcisemer** [Dryec/tron-wallet-android#11]
Further to https://mobile.twitter.com/jcisemer/status/1019324072365383680:

Status Quo:
- Wallet mixes Send/Receive with Freeze (for Voting) and History (non-daily action)
- Freeze tab is disconnected from where you do all the voting (which happens in other part of the app where you pick candidates and cast votes)

Feature Request:
- Transact section - limit to Send/Receive, pull icon into section header, rename section “Transact with Tron wallet”, 
- Freeze tab - move Freeze tab into Voting section, pull icon into section header, rename section “Vote for Super Reps”

In short, make user flows smoother by starting to group tasks by logical user journeys, with everything transactions-related (send/receive) grouped together into on tabbed group, and everything voting-related (freeze, candidates, your votes) grouped together into another tabbed group.

![vote-transact-large](https://user-images.githubusercontent.com/10479630/42845087-e41f22d8-89e2-11e8-98d5-3edad42b5f28.png)
